src/components/set-yxz-order.js

```diff
@@ -0,0 +1,5 @@
+AFRAME.registerComponent("set-yxz-order", {
```

Comment:
Ho boy, I know why we might want this but it scares me to have another thing doing this... I guess its better than adding a rotation component just for its side-effects...

Reply:
yeah this is in the "step forward" category but not going to argue that this isn't fundamentally concerning. i didn't know until this week that information loss when converting through Eulers is axis-dependent